### PR TITLE
fix: replace experiment config

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
[WEB-1952]

## Test Plan
- Check if all pages and actions that were related to `experiment.config` should work as before
- Check if the deprecated config is not used anywhere in web

## Commentary (optional)
- Parallel Coord comparison view doesnt display chart even before this PR. It is a known bug.

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

[WEB-1952]: https://hpe-aiatscale.atlassian.net/browse/WEB-1952?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ashtonG** [comment]
re: the parallel coordinate chart -- is there a ticket tracking that bug?

**Comment by keita-determined** [comment]
i made a ticket https://hpe-aiatscale.atlassian.net/browse/WEB-1962

**Comment by ashtonG** [comment]
given that we now fetch the configs of the selected experiments in the parallel coordinate view, make sure we show the graph as loading while those requests are going on.

File: webui/react/src/pages/F_ExpList/CompareParallelCoordinates.tsx [hunk 1]
```diff
@@ -264,7 +291,7 @@ const CompareParallelCoordinates: React.FC<Props> = ({
     });
   }, [selectedExperiments, selectedMetric, fullHParams, metricData, selectedScale, trials, data]);
 
-  if (!isLoaded) {
+  if (!isLoaded || isFetching) {
```
Comment (ashtonG) [inline webui/react/src/pages/F_ExpList/CompareParallelCoordinates.tsx:294]: state isn't necessary here -- just use whether or not the loadable is loading

Comment (keita-determined) [reply webui/react/src/pages/F_ExpList/CompareParallelCoordinates.tsx:294]: thats so true. will fix it shortly

Comment (keita-determined) [reply webui/react/src/pages/F_ExpList/CompareParallelCoordinates.tsx]: fixed

**Review by ashtonG — APPROVED** [review]

**Pull request merged**